### PR TITLE
Fixed sampling rate issue - batchEvent

### PR DESCRIPTION
**Pull request by qmjava**
Fixed sampling rate issue
1.worker thread and appconfig thread are not starting correctly
2. appconfig is not able to get updated due to the null etag check
3. only update appconfig when etag is different
4. prevent race condition of appConfig
5. bump up version

### CODE REVIEW

File: Moesif.Middleware/Helpers/AppConfigHelper.cs [hunk 1]
```diff
@@ -13,16 +13,29 @@ namespace Moesif.Middleware.Helpers
...
-            AppConfig appConfig = prevAppConfig;
 
             try
             {
                 var appConfigResp = await client.Api.GetAppConfigAsync();
-                appConfig = ApiHelper.JsonDeserialize<AppConfig>(appConfigResp.Body);
-                appConfig.etag = appConfigResp.Headers.ToDictionary(k => k.Key.ToLower(), k => k.Value)["x-moesif-config-etag"];
-                appConfig.lastUpdatedTime = DateTime.UtcNow;
+                var etag = appConfigResp.Headers.ToDictionary(k => k.Key.ToLower(), k => k.Value)["x-moesif-config-etag"];
+                if (etag != config.etag)
```
Comment (praves77) [inline Moesif.Middleware/Helpers/AppConfigHelper.cs:32]: @qmjava How does it guarantee that an old config will not be used to override current config?

File: Moesif.Middleware/Moesif.Middleware.nuspec [hunk 2]
```diff
@@ -2,7 +2,11 @@
 <package >
   <metadata>
     <id>Moesif.Middleware</id>
+<<<<<<< HEAD
```
Comment (praves77) [inline Moesif.Middleware/Moesif.Middleware.nuspec:5]: needs fix

Comment (qmjava) [reply Moesif.Middleware/Moesif.Middleware.nuspec:5]: yep

**Review by praves77 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -711,6 +711,7 @@ Moesif has validated `Moesif.Middleware` against the following framework.
 |                | Framework Version  |
 | -------------- | -----------------  | 
 | .NET |5.0|
+| .NET |6.0|
```
Comment (praves77) [inline README.md:714]: I would put it on the top.

**Pull request merged**